createdisk.sh

```diff
@@ -136,8 +159,12 @@ libvirtDestDir="${destDirPrefix}_libvirt_${destDirSuffix}"
 rm -fr ${libvirtDestDir} ${libvirtDestDir}.crcbundle
 mkdir "$libvirtDestDir"
 
-create_qemu_image "$libvirtDestDir" "${VM_PREFIX}-base" "${VM_NAME}"
-mv "${libvirtDestDir}/${VM_NAME}" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
+if [ $BUNDLE_TYPE != "microshift" ]; then
+    create_qemu_image "$libvirtDestDir" "${VM_PREFIX}-base" "${VM_NAME}"
+    mv "${libvirtDestDir}/${VM_NAME}" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
+else
+    create_sparsify_qemu_image "${libvirtDestDir}"
```

Comment:
`create_qemu_image` already creates a sparse qemu image, so it's quite confusing what's being done here.

Reply:
the one we use have different use since there we commit a backup image to master and then sparsify it but here we don't do any of that and just need to sparsify it.